### PR TITLE
streamsets-datacollector 3.3.0 (new formula)

### PR DESCRIPTION
**Pull request by abruneau**
Install a local Streamsets Data Collector on default port 18630

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/streamsets-datacollector.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
+class StreamsetsDatacollector < Formula
+  desc "DataOps Platform for Modern Data Movement"
+  homepage "https://streamsets.com"
+  url "https://archives.streamsets.com/datacollector/3.3.0/tarball/streamsets-datacollector-core-3.3.0.tgz"
+  sha256 "c980e4ecca091c8be2d7769d5c0a4a469e8a2096d1f70b1d502536520d6f980f"
+
+  bottle :unneeded
+  depends_on :java => "1.8+"
```
Comment (apjanke) [inline Formula/streamsets-datacollector.rb]: Do you know if streamsets-datacollector is supported under Java 9 and 10? (It looks like it uses low-level stuff that may run in to issues there.) If it's not, may want to set this to exactly `"1.8"`.

File: Formula/streamsets-datacollector.rb [hunk 2]
```diff
@@ -0,0 +1,59 @@
+class StreamsetsDatacollector < Formula
+  desc "DataOps Platform for Modern Data Movement"
+  homepage "https://streamsets.com"
+  url "https://archives.streamsets.com/datacollector/3.3.0/tarball/streamsets-datacollector-core-3.3.0.tgz"
+  sha256 "c980e4ecca091c8be2d7769d5c0a4a469e8a2096d1f70b1d502536520d6f980f"
+
+  bottle :unneeded
+  depends_on :java => "1.8"
```
Comment (DomT4) [inline Formula/streamsets-datacollector.rb]: Can we leave a new line between `bottle :unneeded` and the first `depends_on`? Thanks.

File: Formula/streamsets-datacollector.rb [hunk 3]
```diff
@@ -0,0 +1,60 @@
+class StreamsetsDatacollector < Formula
+  desc "DataOps Platform for Modern Data Movement"
+  homepage "https://streamsets.com"
+  url "https://archives.streamsets.com/datacollector/3.3.0/tarball/streamsets-datacollector-core-3.3.0.tgz"
+  sha256 "c980e4ecca091c8be2d7769d5c0a4a469e8a2096d1f70b1d502536520d6f980f"
+
+  bottle :unneeded
+
```
Comment (DomT4) [inline Formula/streamsets-datacollector.rb]: Some trailing whitespace here now, which needs snipping.

Comment (abruneau) [reply Formula/streamsets-datacollector.rb]: Sorry, I'll try to get it right this time

Comment (DomT4) [reply Formula/streamsets-datacollector.rb]: No worries! It's usually a case of text editors _trying_ to be helpful but not quite managing it. Thanks for adding the line 😺

File: Formula/streamsets-datacollector.rb [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+  depends_on :java => "1.8"
+  depends_on "md5sha1sum"
```
[the remark this answers is not in the document]
Comment (abruneau) [reply Formula/streamsets-datacollector.rb]: I just made a pull request with you suggestion

[the remark this answers is not in the document]
Comment (apjanke) [reply Formula/streamsets-datacollector.rb]: Could you include a link to your PR?

[the remark this answers is not in the document]
Comment (DomT4) [reply Formula/streamsets-datacollector.rb]: https://github.com/streamsets/datacollector/pull/64

[the remark this answers is not in the document]
Comment (abruneau) [reply Formula/streamsets-datacollector.rb]: According to the last comment, I don't think Streamsets will change it's code

[the remark this answers is not in the document]
Comment (apjanke) [reply Formula/streamsets-datacollector.rb]: Okay. No big deal, IMHO.

**Comment by abruneau** [comment]
Did I miss something to merge my branch?

**Comment by DomT4** [comment]
@ilovezfs Probably another one for you to look over this. IMO the `md5sha1sum` thing doesn't need to be a blocker, given we've permitted it elsewhere, for better or worse.

**Pull request closed without merging**